sdk/authorization/arm-authorization/src/models/index.ts

```diff
@@ -1,781 +1,624 @@
 /*
  * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import { BaseResource, CloudError, AzureServiceClientOptions } from "@azure/ms-rest-azure-js";
 import * as msRest from "@azure/ms-rest-js";
 
 export { BaseResource, CloudError };
 
-
 /**
- * @interface
- * An interface representing ClassicAdministrator.
  * Classic Administrators
- *
  */
 export interface ClassicAdministrator {
   /**
-   * @member {string} [id] The ID of the administrator.
+   * The ID of the administrator.
    */
   id?: string;
   /**
-   * @member {string} [name] The name of the administrator.
+   * The name of the administrator.
    */
   name?: string;
   /**
-   * @member {string} [type] The type of the administrator.
+   * The type of the administrator.
    */
   type?: string;
   /**
-   * @member {string} [emailAddress] The email address of the administrator.
+   * The email address of the administrator.
    */
   emailAddress?: string;
   /**
-   * @member {string} [role] The role of the administrator.
+   * The role of the administrator.
```

Comment:
Are these changes generated or hand made?

Reply:
generated i guess

Reply:
Yeap. This is generated